accepted/2021/utf8/README.md

```diff
@@ -0,0 +1,21 @@
+# Improve UTF-8 support
+
+**DRAFT**
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+UTF-8 is currently the most popular encoding on the web with [more than 95% of all websites using it](https://w3techs.com/technologies/cross/character_encoding/ranking). Additionally it's used as a default encoding on Linux. When .NET was created there was many character encodings used widely and UTF-8 was not dominant at the time - .NET has chose to use UTF-16. As UTF-8 became more popular it become more obvious that UTF-8 has many memory advantages over other encodings including UTF-16 and therefore its great adoption.
+
+## Benefits of using UTF-8 over UTF-16
+
+- Reduces heap allocations - in many apps the number of memory needed for storing text is expected to be reduced twice
+- Improves quality, readability, and maintainability of code - performance oriented apps frequently store UTF-8 strings as byte arrays or workaround the platform limitations in other ways
```

Comment:
At the inconvenience of UTF-16 code now dropping in quality, readability and maintainability because of the need to do workarounds being transferred to them.

Reply:
Adding this to the drawbacks section